File: song.java

import java.util.*;
/*
 * @author Andy Phan
 */
public class song {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        
        int stanzas = in.nextInt(); //read in number of stanzas
        
        for(int i = 1; i <= stanzas; i++) {
            int repeat = in.nextInt(); //gets number of times to reapeat
            System.out.printf("Stanza #%d:\n", i);
            
            for(int j = 0; j < repeat; j++) //prints each lyric
                System.out.println("This song's gonna get stuck inside yo'");
            
            System.out.println("head"); //finishes stanza
            System.out.println("");
        }
    }
}

File: song.cpp
#include <iostream>

int main() {
  int cases, lines, i, j;

  std::cin >> cases;

  for (i = 0; i < cases; i++) {
    std::cin >> lines;

    // Output the stanza number.
    std::cout << "Stanza #" << (i+1) << ":\n";

    for (j = 0; j < lines; j++) {
      std::cout << "This song's gonna get stuck inside yo'\n";
    }
    std::cout << "head\n";

    // Follow each case with a blank line.
    std::cout << std::endl;
  }

  return 0;
}
